modules/core/src/main/java/org/apache/ignite/internal/processors/query/running/QueryHistory.java

```diff
@@ -47,13 +47,15 @@ public class QueryHistory {
      * @param startTime Start time of query execution.
      * @param duration Duration of query execution.
      * @param failed {@code True} query executed unsuccessfully {@code false} otherwise.
+     * @param initId Initiator ID.
      */
-    public QueryHistory(String qry, String schema, boolean loc, long startTime, long duration, boolean failed) {
+    public QueryHistory(String qry, String schema, boolean loc, long startTime, long duration, boolean failed,
```

Comment:
Please fix codestyle (in case of multi-line declaration, each parameter should be on it's own line)